src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -628,8 +625,7 @@ private void processLogsAndUpload() throws InterruptedException {
                             if (componentCurrentProcessingLogFile.containsKey(componentName)) {
                                 CurrentProcessingFileInformation processingFileInformation =
                                         componentCurrentProcessingLogFile.get(componentName);
-                                if (processingFileInformation.fileName.equals(file.getAbsolutePath())
-                                        && processingFileInformation.lastModifiedTime == file.lastModified()) {
```

Comment:
Why is this check for lastModifiedTime not needed anymore? - Is it because of the same name rotation?

Reply:
My guess: it was just a double confirmation.